common/src/main/resources/db/changelog/v001/log_event_schema.sql

```diff
@@ -0,0 +1,131 @@
+--liquibase formatted sql
+--  -------------------------------------------------------------------------------------------------------------------
+
+--changeset lsharshar:AB2D-1242-persist-events-api-request failOnError:true
+
+CREATE TABLE event_api_request
+(
+        id SERIAL,
+        time_of_event TIMESTAMP WITH TIME ZONE,
+        job_id VARCHAR(255),
+        user_id VARCHAR(255),
+        url VARCHAR(2048),
+        ip_address VARCHAR(32),
+        token_hash VARCHAR(32),
```

Comment:
What's token hash? How would we use it?

Reply:
I wanted a way to "save" the token so we can see if they are using the same token but not necessarily have to actually save the entire token - we don't know how large it will end up being (larger than useful) and even so, we pick out the needed data in other places like username. I just want to know if they are the same one to answer questions like, are they are using a new one every time, are they using it after it expires? That sort of thing.